usr/src/uts/i86pc/vmm/Makefile

```diff
@@ -11,7 +11,7 @@
 
 #
 # Copyright 2013 Pluribus Networks Inc.
-# Copyright 2019 Joyent, Inc.
+# Copyright 2020 Joyent, Inc.
```

Comment:
Update everything copyright-y to 2021.

Reply:
This code hasn't been touched in over a year, so...